src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfoV2.jsx

```diff
@@ -87,8 +90,14 @@ export default function ConfirmationDirectScheduleInfoV2({
           className="far fa-calendar vads-u-margin-right--1"
         />
         <AddToCalendar
-          summary="VA Appointment"
-          description=""
+          summary={`Appointment at ${clinic.serviceName}`}
```

Comment:
Similar comment to elsewhere, I think this should match the VA in person copy and use facility name instead of clinic name.